orm_lib/inc/drogon/orm/Field.h

```diff
@@ -133,7 +133,7 @@ class DROGON_EXPORT Field
             if (arrVal.first == ArrayParser::juncture::string_value)
             {
                 T val;
-                std::stringstream ss(std::move(arrVal.second));
```

Comment:
@an-tao 
According to [`ArrayParser.h`](https://github.com/drogonframework/drogon/blob/c5e596d942b8c9b51d7444d93811bc5a0c9401b4/orm_lib/inc/drogon/orm/ArrayParser.h#L79), `arrVal.second` is `std::string`.

The move constructor of `std::stringstream` is just for moving `std::stringstream`[^1][^2], not for `std::string`. That's why I believe this `std::move` is misused.

[^1]: https://cplusplus.com/reference/sstream/stringstream/stringstream/
[^2]: https://en.cppreference.com/w/cpp/io/basic_stringstream/basic_stringstream

Reply:
However, using `std::move` does not seem to have any side effects. And if, I mean, technically, stringstream implements a `move string version` of the constructor in the future, this way is beneficial.

Reply:
It looks like C++20 implements that type of constructor, so as you mentioned, it's beneficial in C++20 and later. Thank you.